src/Paprika/Utils/Xor8.cs

```diff
@@ -216,6 +221,10 @@ private int GetHash(ulong key, ulong seed, int index)
 
     private static int Fingerprint(ulong hash) => (int)(hash & ((1 << BitsPerFingerprint) - 1));
 
+    ~Xor8()
```

Comment:
Finalizers are bad for performance, would be better to use IDisposable or similar

Reply:
Yes, will remove it.